sites/partners/src/listings/PaperListingForm/sections/ListingPhotos.tsx

```diff
@@ -0,0 +1,316 @@
+            type="hidden"
+            name={`images[${index}].image.fileId`}
+            ref={register()}
+            defaultValue={item.image.fileId}
+          />
+        </span>
+      ))}
+      <GridSection
+        grid={false}
+        separator
+        title={t("listings.sections.photoTitle")}
+        description={t("listings.sections.photoSubtitle")}
+      >
+        <span className={"text-tiny text-gray-800 block mb-2"}>
+          {t("listings.sections.photoTitle")}
+        </span>
+        <GridSection columns={1} tinted inset>
+          <GridCell>
+            {listingFormPhotos.length > 0 && (
+              <div className="mb-5">
+                <MinimalTable
+                  headers={photoTableHeaders}
+                  data={listingPhotoTableRows}
+                ></MinimalTable>
+              </div>
+            )}
+
+            <Button
+              type="button"
+              styleType={fieldHasError(errors?.images) ? AppearanceStyleType.alert : null}
+              onClick={() => {
+                setDrawerState(true)
+                setDrawerImages([...listingFormPhotos])
+                clearErrors("images")
+              }}
+              data-test-id="add-photos-button"
+            >
+              {t(listingFormPhotos.length > 0 ? "listings.editPhotos" : "listings.addPhoto")}
+            </Button>
+          </GridCell>
+        </GridSection>
+      </GridSection>
+      {fieldHasError(errors?.images) && (
+        <span className={"text-sm text-alert"}>{errors?.images?.nested?.message}</span>
+      )}
+
+      {/* Image management and upload drawer */}
+      <Drawer
+        open={drawerState}
+        title={t(listingFormPhotos.length > 0 ? "listings.editPhotos" : "listings.addPhoto")}
+        onClose={() => resetDrawerState()}
+        ariaDescription="Form with photo upload dropzone"
+      >
+        <section className="border rounded-md p-8 bg-white">
+          <h2 className="text-lg mb-8">{t("listings.listingPhoto")}</h2>
+          {drawerImages.length > 0 && (
+            <div className="mb-10">
+              <span className={"text-tiny text-gray-800 block mb-2"}>{t("t.photos")}</span>
+              <MinimalTable
+                draggable={true}
+                flushLeft={true}
+                setData={(newData) => {
+                  setDrawerImages(
+                    newData.map((item: Record<string, StandardTableCell>, index) => {
+                      const foundImage = drawerImages.find(
+                        (field) =>
+                          field.image.fileId.split("/").slice(-1).join() == item.fileName.content
+                      )
+                      return { ...foundImage, ordinal: index }
+                    })
+                  )
+                }}
+                headers={drawerTableHeaders}
+                data={drawerTableRows}
+              ></MinimalTable>
+            </div>
+          )}
+          {drawerImages.length < 3 ? (
```

Comment:
UI changes look good! I think this just needs to be updated to 10 since it currently limits it to 3 images.